res/values-en/strings.xml

```diff
@@ -52,7 +52,7 @@
     <string name="pref_reset">Reset</string>
     <string name="pref_reset_summary">Use Default Settings to Replace the contents in /sdcard/rime</string>
 
-    <string name="pref_preferences">Preferences</string>
+    <string name="pref_keyboard">Preferences</string>
```

Comment:
Google pinyin is better for Chinese people, as it is more popular than AOSP(except Sougou).